tests/group_tree.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+import sunbeam
+
+
+class TestGroupTree(unittest.TestCase):
+    def setUp(self):
+        norne = '../../examples/data/norne/NORNE_ATW2013.DATA'
+        self.norne = sunbeam.parse(norne, ('PARSE_RANDOM_SLASH', sunbeam.action.ignore))
+
+    def test_group_tree(self):
+        self.gr_obj = self.norne.schedule.group_tree(0)
+        parent = self.gr_obj.parent('PROD')
+        children = self.gr_obj.children('PROD')
+
+        self.assertEqual('FIELD', parent)
+        self.assertEqual('MANI-B1', children[0])
+        self.assertEqual(6, len(children))
+
+    def test_grouptree(self):
+        tree = self.norne.schedule.grouptree(0)
```

Comment:
I don't think it's wise for the `schedule` API to have one `.group_tree` function and one `.grouptree` function.  What will the users think?

Reply:
At least the users can now use both `xxx.['children']` and `xxx.children` like you reqested.  The latter is also available from the private function `xxx._children` when using the `libsunbeam.grouptree` object calling `schedule._group_tree`, so we can possibly remove `.group_tree`?